docs/sdk/server-side-sdks/python/python-usage.md

```diff
@@ -171,3 +171,26 @@ To further enable various deployment configurations - we provide a background pr
 or the SDK is not able to make requests to the DevCycle API directly. The installation and setup process for the proxy can be found here: https://github.com/DevCycleHQ/sdk-proxy#readme.
 
 See the [SDK Proxy](../../sdk-proxy/index.md) section for more information.
+
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
```

Comment:
wouldn't the more relevant user-facing benefit of this be that flag updates are delivered more quickly? This all sounds like technical detail that mostly we care about internally

Reply:
@ajwootto polling every second means the speed of realtime updates is less relevant on server-side. The biggest benefit server-side is I can dial-down polling while still getting updates delivered quickly.